### ISSUE TITLE
cannot find any .npy file of  Dataset or generation method

### ISSUE THREAD

**Issue by littlePrince126** [YadiraF/DECA#77]
Thanks for your nice work!

Could you please provide a list of training data? 
or tell us how to get it,  eg.(vggface2_val_list_max_normal_100_ring_5_1_serial.npy)

Thanks a lot!

**Comment by Ned09** [comment]
@littlePrince126 
Any chance to get this file? "vggface2_train_list_max_normal_100_ring_5_1_serial.npy"

**Comment by Shelomith** [comment]
any one solved this problem?

**Comment by littlePrince126** [comment]
The voxceleb2 dataset have more than 1million videos. I have no idea how to choose 500k images.

**Comment by Shelomith** [comment]
> The voxceleb2 dataset have more than 1million videos. I have no idea how to choose 500k images.

Can we just randomly select 500k images? from each identity? Some paper extract front or side view frame from video.

**Comment by littlePrince126** [comment]
@Shelomith Yes, I agree with you. I used an IQA model to choose good quality videos.

**Comment by zydmu123** [comment]
@littlePrince126 Hello, here is a question. In the process of data cleaning, it is said that "generate a different crop for the face image by shifting the provided bounding box by 5%..."in the Supplemental. However, as I know, there is no the label of bbox for the BUPT_Balancedface and the VoxCeleb2 datasets, I wonder how did you handle it, thanks a lot!

**Comment by prokaryote-hgy** [comment]
> Thanks for your nice work!
> 
> Could you please provide a list of training data? or tell us how to get it, eg.(vggface2_val_list_max_normal_100_ring_5_1_serial.npy)
> 
> Thanks a lot!

Hi, have you got the file vggface2_val_list_max_normal_100_ring_5_1_serial.npy and other similar files? I have no idea how to prepare the training data, could you please offer me some help?

**Comment by prokaryote-hgy** [comment]
> Thanks for your nice work!
> 
> Could you please provide a list of training data? or tell us how to get it, eg.(vggface2_val_list_max_normal_100_ring_5_1_serial.npy)
> 
> Thanks a lot!

@Shelomith Hi, have you got the file vggface2_val_list_max_normal_100_ring_5_1_serial and other similar files? I have no idea how to prepare the training data, could you please offer me some help?

**Comment by prokaryote-hgy** [comment]
@littlePrince126 Does VGGFace2 only select 5 images per individual in the dataset? With approximately 9,000 images in VGGFace2, if each person only has 5 pictures chosen, there would only be a total of 45,000 images available. However, the paper claims to have used 950,000 face images from VGGFace2.

> Just want to know how to choose the 5 max normal faces of every identity.
